src/validate-changelog.ts

```diff
@@ -107,7 +107,7 @@ type ValidateChangelogOptions = {
  * `true` and the changelog contains uncategorized changes.
  * @throws `ChangelogFormattingError` - Will throw if there is a formatting error.
  */
-export function validateChangelog({
+export async function validateChangelog({
```

Comment:
Nit:  `README` should be updated with the new API (though, I believe it should be updated with other things as well i.e. node version)

Reply:
Will do in a separate PR.